docs/proposed/http-storage-node-protocol.rst

```diff
@@ -630,11 +630,14 @@ Reading
 ``GET /v1/immutable/:storage_index/shares``
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-Retrieve a list indicating all shares available for the indicated storage index.
-For example::
+Retrieve a list (semantically, a set) indicating all shares available for the
+indicated storage index. For example::
 
   [1, 5]
 
+An unknown storage index results in empty list, so that lack of existence of
+storage index is not leaked.
+
```

Comment:
I'm not sure if "unknown storage index" and "storage index that doesn't exist" are actually distinct things.  "Return an empty list" seems like the right behavior to me though.